mxcube3/routes/samplecentring.py

```diff
@@ -112,7 +112,7 @@ def move_to_centred_position(point_id):
 
 
 @server.route("/mxcube/api/v0.1/sampleview/shapes", methods=["GET"])
-@server.restrict
+#@server.restrict
```

Comment:
why unrestrict?

Reply:
Opps, sorry yes it should be restricted :) Something left out while testing.